### PR TITLE
Bugfix MTE-3921 Use previous OS version in Firebase perf tests

### PR DESCRIPTION
**Pull request by isabelrios**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-3921)

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -1572,6 +1572,10 @@ workflows:
             sed 's/.*=//;s/'\''/"/g' data.txt > test.json
             cat test.json
             python3 perfTestTransform.py
+    meta:
+      bitrise.io:
+        stack: osx-xcode-15.4.x
```
Comment (isabelrios) [inline bitrise.yml:1577]: @clarmso would we be able to build using previous stack? I know we did this for L10nBuild workflow so trying same approach

File: bitrise.yml [hunk 2]
```diff
@@ -1432,6 +1431,7 @@ workflows:
                 xcodebuild -showsdks | grep -E 'iphoneos|iphonesimulator'
                 exit 1
             fi
+    - deploy-to-bitrise-io@2.9.2: {}
```
Comment (isabelrios) [inline bitrise.yml:1434]: Moving the deploy step below the Build for testing to also have the xctestrun file in the Artifacts

File: bitrise.yml [hunk 3]
```diff
@@ -1379,8 +1379,7 @@ workflows:
         - output_tool: xcodebuild
         - scheme: Fennec_Enterprise
         - configuration: Fennec_Enterprise
-    - deploy-to-bitrise-io@2.9.2: {}
-    - script@1.1:
+    - script@1.2.1:
```
Comment (isabelrios) [inline bitrise.yml:1382]: Upgrading the script step

File: bitrise.yml (hunk 1 above)
Comment (isabelrios) [reply bitrise.yml:1577]: we can not test this workflow because the IPA is only generated from main, not from PRs

Comment (clarmso) [reply bitrise.yml:1577]: No, the current change in `main` is not backwards compatible.

Comment (isabelrios) [reply bitrise.yml:1577]: the L10nBuil is building successfully with this approach 🤔  can we give this a try? This will only change the perf workflow

Comment (clarmso) [reply bitrise.yml:1577]: No, the compilation failed but Bitrise's status is green. 🤔 
https://app.bitrise.io/build/8ad12ec5-3992-4b77-8ebc-01921b9ef345?tab=log

Comment (isabelrios) [reply bitrise.yml:1577]: we should fix that.. if that script step fails, the notification should be red not green :(

**Comment by isabelrios** [comment]
There is not backward compatibility when building so closing this PR as it will not work. The Perf test workflow is disabled for now :(

**Pull request closed without merging**